packages/esm-patient-test-results-app/src/overview/common-overview.tsx

```diff
@@ -126,14 +142,22 @@ const CommonOverview: React.FC<CommonOverviewProps> = ({
           const allNormalResults = !data.some((result) => abnormalInterpretation.includes(result.interpretation));
           const patientSummaryDashboardData = data.slice(0, DashboardResultsCount);
           return (
-            <Card allNormalResults={allNormalResults} key={uuid}>
+            <Card
+              allNormalResults={allNormalResults}
+              key={uuid}
+              isPatientSummaryDashboard={isPatientSummaryDashboard}
```

Comment:
Yikes, I know this is preexising but `isPatientSummaryDashboard` smells fishy to me. That's wayyy too much knowledge for an extension to have about the outside world. What's the actual semantic meaning, within the extension itself? It it like a more minimal form of the extension, or a more compact view, or a mode in which it is able to appear on a dashboard alongside other widgets?